### ISSUE TITLE
Fix wording when authorization token is missing

### ISSUE THREAD

**Issue by serenamarie125** [redhat-developer/rh-che#1431]
**Description:**
Currently when I try to get into Che from OpenShift.com, I see an error message 
"Authorization token is missed."

See this animated gif: http://recordit.co/F2UEY7Ve5q

Can we reword this to "Authorization token is missing. Reload the page." & make the entire _Reload the page_ string be a link?

FYI @davidfestal - I heard that the UI looks for a token in the local storage and mentions the error message as is, and that you are the developer who can fix the wording for us.

**Comment by ibuziuk** [comment]
@serenamarie125 do you happen to know what was the root cause of the failure. Does the info in FAQ helps with your specific issue https://github.com/redhat-developer/rh-che/blob/master/FAQ.adoc#can-not-login-to-cheopenshiftio---authorization-token-is-missed ?

**Comment by serenamarie125** [comment]
@ibuziuk I just tested using chrome, and it works fine.  The issue is when I'm using Safari.  I'll check the FAQ and report back with more detail.

**Comment by ibuziuk** [comment]
chrome is a recommended browser. Closing this as duplicate in favour of upstream one https://github.com/eclipse/che/issues/11817

**Comment by ppitonak** [comment]
@ibuziuk upstream issue is about the token itselft, this one is about changing the error message, you should probably reopen or update the upstrem one.
